### ISSUE TITLE
HUB75 pinout

### ISSUE THREAD

**Issue by Danijel777** [hzeller/rpi-rgb-led-matrix#937]
I recently acquired 18 RGB 32x64 panels for a video project at school. The supplier claims to be the HUB75 standard.
The pin layout is as pictured. Twice the pin R1 is specified. Also no line D. Four lines are marked N (I assume it's GND). Has anyone encountered this kind of pin layout. Or is the pin layout wrong?

![IMG_20191212_121150](https://user-images.githubusercontent.com/57591178/70758991-0ca04380-1d45-11ea-9506-182665f68d78.jpg)

**Comment by Danijel777** [comment]
II misspelled. Resolution is 32 x 64 pixells.

**Comment by hzeller** [comment]
It just looks like a typo on the silk-screen, so the second `R1` meant `G2`.

It can be only A, B, C without D if this is an outdoor panel (but given that there already is a typo on the silk-screen, who knows :) ).

**Comment by Danijel777** [comment]
Thanks a lot. Today I will try to connect.

pet, 13. pro 2019. 05:51 Henner Zeller <notifications@github.com> je
napisao:

**Comment by Danijel777** [comment]
Problem solved!
Yes, this is an outdoor panel. Typo is the second R1 (this is G2).

What I did:
First, I didn't connect the D wire.
Secondly, I needed to adjust the parameters to display the content.
This unique matrix (64 x 32) is organized in two  32 x 32 matrix. So, I had to put --led-chain = 2

Other parameters I changed:

- --led-row-addr-type = 3
- --led-multiplexing = 2
- - follows-slow-gpio = 2

Complete statement is:
**sudo ./demo --led-cols = 32 --led-rows = 32 --led-slowdown-gpio = 2 --led-multiplexing = 2 -led-row-addr-type = 3 -led-chain = 2 -led-brightness = 100 -D 1 runtext.ppm**

The only minor problem is the flickering of some pixels. I attribute this to the interference between data wires. 

Thank Mr. Zeller for good tips!

Next step is building small video wall of 3x5 matrix. In few days I expect delivering Adafruit RGB Matrix HAT. 

https://youtu.be/iz5QsZ67Mqw
https://youtu.be/Qy9L6IfF51Q


![IMG_20191213_233513](https://user-images.githubusercontent.com/57591178/70838347-163eaf80-1e08-11ea-82e8-af169cd29c3f.jpg)
![IMG_20191213_234521](https://user-images.githubusercontent.com/57591178/70838352-19d23680-1e08-11ea-8d73-6d8b14ff6996.jpg)
